tests/TestyMcTestFace.cpp

```diff
@@ -18,8 +20,19 @@ class FrameIsdTest : public ::testing::Test {
       json jsonIsd = json::parse(isdFile);
       isd.clearAllParams();
```

Comment:
Is the setUp rerun each time and it is necessary to clear the ISD before a second run?

Reply:
setUp is called before every test that uses the framework. We actually don't need to clear the ISD